tool/check-fixme

```diff
@@ -0,0 +1,82 @@
+#!/bin/sh
+# tool/check-fixme -- look for FIXME task labels in the MPS source tree
+# Richard Brooksby, Ravenbrook Limited, 2023-01-15
+#
+# Copyright (c) 2023 Ravenbrook Limited. See end of file for license.
+#
+# Developers can mark code in the MPS source tree with label like
+# "FIXME" and "TODO" in order to leave notes for later work.  These
+# labels are recognized by tools like Xcode.
+#
+# This script finds FIXMEs in code so that we can:
+# 1. fix 'em
+# 2. cause them to fail CI
+# 3. prevent them leaking into master and versions
+# 4. thereby making them useful for communication with self or other devs
+#
+# TODO: A much older convention in the MPS (predates the adoption of
+# "FIXME") is to mark such things with "@@@@".  We should find and fix
+# those too, eventually.
+
+# Exclude some things that:
+# - we have no control over
+# - come from elsewhere
+# - build results that are copies of other things
+# - things that legit talk about FIXMEs (like this script)
```

Comment:
m. rule.generic.self: the command excludes backups too

Reply:
Fix: Added comment about local editor backups in 3f46141e3f1ca46f64cb8f9bb3b8e38f715bee8a